students/jinsu/users/views.py

```diff
@@ -1,3 +1,50 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+from users.models import User
```

Comment:
```suggestion
from django.http  import JsonResponse
from django.views import View

from users.models import User
```
장고 자체 모듈과 내가 생성한 모듈은 분리해주세요!

Reply:
위 수정사항 반영하고 알려주신대로 try/except 구조 수정하였습니다!